modules/import/csv_file.py

```diff
@@ -139,8 +140,7 @@ def __parse_file(self):
             # get each line from the file and separate it into columns based on sep
             for row in csvreader:
                 # append all lines as-is case-wise
-                # unicode(str, errors='ignore') causes all invalid characters to be stripped out
-                values.append([str(value.strip(), errors='ignore') for value in row])
+                values.append([value.strip() for value in row])
```

Comment:
The `unicode()` call was needed in python 2. Somewhere along the line it got converted to `str()` but with Python 3 I'm not sure this is needed as the doc for the csv module says:
> Each row read from the csv file is returned as a list of strings.